### PR TITLE
[New] Add algebra.js w/ git auto-update

### LINKED ISSUE #7971: [Request] algebra.js
**Library name:** algebra.js
**Git repository url:** https://github.com/nicolewhite/algebra.js
**npm package url(optional):** https://www.npmjs.com/package/algebra.js
**License(s):** MIT
**Official homepage:** http://algebra.js.org
**Wanna say something? Leave message here:** They dont appear to currently have a webpack/browserified version in github, but they do offer one [here](http://algebra.js.org/javascripts/algebra.min.js) as well as [make support to build one](https://github.com/nicolewhite/algebra.js#in-the-browser-1).
# 

Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sufuf3**
Close #8152 and move to here as the owner [mention](https://github.com/cdnjs/cdnjs/pull/8152#issuecomment-345315764)

Close #7971
Pull request for issue: #7971
Related issue(s): # #8152

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/nicolewhite/algebra.js
- Official website (optional, not the repository): http://algebra.js.org
- NPM package url (optional): https://www.npmjs.com/package/algebra.js
- GitHub / Bitbucket popularity (required):
  - Count of stars: 1110
  - Count of watchers: 41
  - Count of forks: 60
- NPM download stats (optional):
  - Downloads in the last day: 4
  - Downloads in the last week: 100
  - Downloads in the last month: 589
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [ ] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
1. Add this lib from GitHub because the file can be used in front-end is from git repo. Although, there are more versions from npm source, but the files we can't use in front-end is from npm.
2. Manually add the files of the old versions from https://github.com/nicolewhite/algebra.js/tree/master/build (in master branch not git tag)
3. The detail discuss is #8152

@cdnjs/library-reviewer please help me review this PR, thank you.

File: ajax/libs/algebra.js/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "license": "MIT",
+  "homepage": "http://algebra.js.org",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nicolewhite/algebra.js.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "**/*"
```
Comment (extend1994) [inline ajax/libs/algebra.js/package.json]: https://github.com/nicolewhite/algebra.js/tree/master/build
Seems like all filenames including version number and all versions would be fetched for now. Maybe don't set up auto-update config for now is a better idea?

Comment (sufuf3) [reply ajax/libs/algebra.js/package.json]: As this discussion https://github.com/cdnjs/cdnjs/pull/8152#issuecomment-295075986 I think it's fine here.
What do you think?
Thanks.

**Comment by sufuf3** [comment]
@extend1994 squash the commits in one commit just manually add these versions from master branch of git repo.
And I think add git auto-update is fine here, because the author will do that as pvnr0082t suggestion.
https://github.com/cdnjs/cdnjs/pull/12379#discussion_r165810131
What do you think?
Thanks.

File: ajax/libs/algebra.js/package.json (hunk 1 above)
Comment (extend1994) [reply ajax/libs/algebra.js/package.json]: OK

**Pull request merged**